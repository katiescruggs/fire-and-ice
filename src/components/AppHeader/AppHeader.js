import React from 'react';
import logo from '../App/logo.svg';

const AppHeader = () => (
  <div className='App-header'>
    <img src={logo} className='App-logo' alt='logo' />
    <h2>Welcome to Westeros</h2>
  </div>);

export default AppHeader;