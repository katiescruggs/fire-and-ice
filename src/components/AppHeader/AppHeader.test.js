import React from 'react';
import { shallow } from 'enzyme';
import AppHeader from './AppHeader';

describe('AppHeader', () => {
  let appHeader;
  beforeEach(() => {
    appHeader = shallow(<AppHeader />);
  });

  it('should exist', () => {
    expect(appHeader).toBeDefined();
  });

  it('should match snapshot', () => {
    expect(appHeader).toMatchSnapshot();
  });
});