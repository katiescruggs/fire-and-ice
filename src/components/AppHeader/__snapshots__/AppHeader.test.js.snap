// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AppHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="logo"
          className="App-logo"
          src="logo.svg"
/>,
        <h2>
          Welcome to Westeros
</h2>,
      ],
      "className": "App-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "logo",
          "className": "App-logo",
          "src": "logo.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome to Westeros",
        },
        "ref": null,
        "rendered": "Welcome to Westeros",
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="logo"
            className="App-logo"
            src="logo.svg"
/>,
          <h2>
            Welcome to Westeros
</h2>,
        ],
        "className": "App-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "logo",
            "className": "App-logo",
            "src": "logo.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to Westeros",
          },
          "ref": null,
          "rendered": "Welcome to Westeros",
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
